spec/lang/representation.md

```diff
@@ -99,7 +99,9 @@ When decoding, we have to deal with the possibility of the pointer bytes not all
 this is define to yield a pointer without provenance.
 On the other hand, some pointers are wide pointers which also need to encode their metadata.
 The helpers `decode_ptr` and `encode_ptr` deal with the thin pointers,
-and the `decode_ptr_meta` helper inferrs the metadata kind from the pointee layout.
+and the `decode` call them when the pointer is thin.
```

Comment:
I can't quite parse this... should it be "calls them"? And no "the"?
But it also seems unnecessary to say this, the `decode` sources are right below. Saying they deal with thin pointers conveys all the relevant high-level parts, IMO.